zojae031/Project/app/src/main/java/study/architecture/ui/coinjob/CoinDataAdapter.kt

```diff
@@ -18,16 +16,21 @@ class CoinDataAdapter : RecyclerView.Adapter<CoinDataAdapter.Holder>(),
     CoinAdapterContract.Model {
 
     private val lists: MutableList<ProcessingTicker> = mutableListOf()
-
-    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): Holder =
-        Holder(LayoutInflater.from(parent.context).inflate(R.layout.list_item, parent, false))
+    private lateinit var binding: ListItemBinding
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): Holder {
+        binding = ListItemBinding.inflate(
+            LayoutInflater.from(parent.context), parent, false
```

Comment:
아래와 같이 정리하시는 것이 좋아 보입니다.
```suggestion
            LayoutInflater.from(parent.context), 
            parent, 
            false
```
https://kotlinlang.org/docs/reference/coding-conventions.html#method-call-formatting

Reply:
[e180909](https://github.com/StudyFork/GoogryAndroidArchitectureStudy/pull/194/commits/e1809098c3f0dcf9e95831dbdfeabc51d99165e5)